src/components/DropListButton.js

```diff
@@ -0,0 +1,15 @@
+import "./DropListButton.css"
+import dropDown from "../assets/images/dropDown.svg"
+
+const DropListButton = (props) => {
+    return (
+        <button onClick={() => { props.onChangeVisibilityOfDopList() }}
```

Comment:
`<button onClick={props.onChangeVisibilityOfDopList}` 
should be enough

Reply:
yes , thx